integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/common/EndPointParser.java

```diff
@@ -11,20 +13,37 @@ public class EndPointParser {
     private EndPointParser() {
     }
 
-    public static Optional<String> getEndpointBaseUrl(String endpoint) {
+    public static Optional<String> getBaseUrl(String endpoint) {
         Matcher matcher = ENDPOINT_URL_REGEX.matcher(endpoint);
```

Comment:
One optional idea:
As we have a functionality to parse whole URL now, would it have sense to drop the regex and retrieve all needed info directly from URL object? 
We could drop this whole class and process the URL in `BridgeContext`.
Or event store the URL in `BridgeContext` and construct the needed value in `BridgeEndpointBaseResolver`.

Reply:
agree, thanks for the suggestion.